### ISSUE TITLE
A better UI to the full use of editor space

### ISSUE THREAD

**Issue by xfanplus** [marktext/marktext#3218]
Now the UI with multiple tabs:
![0GdlBb33eB-20220416125940061-539](https://user-images.githubusercontent.com/8945502/163662265-c1b896d7-692e-4c85-8cb5-35974c6785f8.png)

You can see that the title bar and the vertical spacing waste a lot of editor space! And the title in the title bar and in the tabs are redundant!

And a better solution may be like this:
![QfWZbYRfya-20220416125841570-537](https://user-images.githubusercontent.com/8945502/163662320-01a73ae9-fa34-408a-bbbb-545324fcc747.png)

modifications:

1. remove the title bar;
2. tabs take the top position, and the space in the right part of tabs is used for dragging;
3. the unsaved dots are moved to the left side of each tab so that the close icon will not block it;

Thanks for your excellent work!

**Comment by YalandHong** [comment]
Some issues related to native title bar and menu: #1238 #1847 #2866

**Comment by mangoepic** [comment]
![image](https://user-images.githubusercontent.com/71931749/163672543-baac63e9-85b4-452c-89a0-386135337232.png)

If you are using a home folder in the sidebar the nav bar does have a use, showing the path

**Comment by mangoepic** [comment]
If you are using a home folder in the sidebar it can show you the path 
![image](https://user-images.githubusercontent.com/71931749/163672574-ee07366c-76aa-4826-9112-6dd51337d723.png)

**Comment by xfanplus** [comment]
> Some issues related to native title bar and menu: #1238 #1847 #2584 #2866

Yes, #1847 and #2866 are actually discussing the same topic. As for now (2022), the default UI of MarkText is custom CSD. So, there should be no problem slimming the UI. 

I have already customized the UI by myself (through pure CSS), and it feels much better. I think others may also enjoy this modification.


![HJaLfYi975](https://user-images.githubusercontent.com/8945502/163679098-ce2d1298-99af-4c23-b3dc-00bfbbfb375e.gif)

**Comment by xfanplus** [comment]
> If you are using a home folder in the sidebar it can show you the path ![image](https://user-images.githubusercontent.com/71931749/163672574-ee07366c-76aa-4826-9112-6dd51337d723.png)

Yes, I'm aware of this feature. But I think the most important part of an editor is editing.

For me, without the title bar, the writing experience is better. And the full file path is not much important, and I can get this information from the sidebar(with a tree-structure path), which means the file path is also redundant.
